starknet-replay/src/histogram/mod.rs

```diff
@@ -0,0 +1,281 @@
+//! This module is responsible for rendering and saving the SVG image of the
+//! libfunc histogram.
+//!
+//! It uses the `plotter` library to generate the histogram
+//! and rendering.
+//!
+//! SVG has been chosen because, as a vector graphics format, it makes reading
+//! text easy and zooming doesn't degrade the quality.
+//!
+//! The file `mod.rs` contains the public interface. The main way to use this
+//! module is by calling the function `export` to render and save the SVG image.
+//! The file `plot.rs` contains the call to the `plotters` library.
+
+use std::ops::{Add, Div};
+use std::path::PathBuf;
+
+use super::runner::replay_statistics::ReplayStatistics;
+use crate::error::HistogramError;
+use crate::histogram::plot::render;
+
+mod plot;
+
+/// This alias improves readability of the histogram parameters.
+type Pixel = u32;
```

Comment:
This comment is uninformative, but also I dislike this alias. A `Pixel` being a `u32` doesn't make sense to me. Additionally you use it as a _count_ of pixels, so if anything `PixelCount` would be a far better name.